app/src/main/java/com/android/solvit/MainActivity.kt

```diff
@@ -316,7 +318,7 @@ fun ProviderUI(
       ModifyProviderInformationScreen(
           listProviderViewModel, authViewModel, locationViewModel, navigationActions)
     }
-    composable(Route.NOTIFICATIONS) {
+    composable(Screen.NOTIFICATIONS) {
```

Comment:
Shouldn't this be considered as an Route rather than Screen

Reply:
Actually in our current implementation there is no difference, this is a navigation coherency problem to be fixed